pkg/render/packet_capture_api.go

```diff
@@ -66,6 +69,9 @@ type PacketCaptureApiConfiguration struct {
 	TrustedBundle               certificatemanagement.TrustedBundle
 	ClusterDomain               string
 	ManagementClusterConnection *operatorv1.ManagementClusterConnection
+
+	// Whether or not the cluster supports pod security policies.
```

Comment:
nit: whether or not can be shortened to whether

Reply:
This comment is copied from other places. I would like to make it consistent with other places for now.